jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/ChangeConnectionSettingsMenu.kt

```diff
@@ -158,7 +159,21 @@ internal class ChangeRegionAction(private val region: AwsRegion) : ToggleAction(
 
     override fun setSelected(e: AnActionEvent, state: Boolean) {
         if (state) {
+            val oldRegion = getAccountSetting(e).selectedRegion
+
             getAccountSetting(e).changeRegion(region)
+
+            if (oldRegion?.partitionId != region.partitionId) {
+                AwsTelemetry.setPartition(
+                    success = true,
+                    partitionid = region.partitionId
+                )
+            }
+
+            AwsTelemetry.setRegion(
+                success = true,
```

Comment:
Hm I guess success really makes no sense here since we don't validate at this point, can we update toolkit-common to remove `result`?

Reply:
Ya, ill remove upstream

Reply:
https://github.com/aws/aws-toolkit-common/pull/65

Reply:
Updated